zerver/tests/test_events.py

```diff
@@ -3193,6 +3194,19 @@ def test_do_delete_first_message_in_stream(self) -> None:
             is_legacy=False,
         )
 
+    def test_check_update_all_streams_active_status(self) -> None:
+        hamlet = self.example_user("hamlet")
+        self.subscribe(hamlet, "test_stream1")
+        stream = get_stream("test_stream1", self.user_profile.realm)
+
+        # Delete all messages in the stream.
+        Message.objects.filter(recipient__type_id=stream.id, realm=stream.realm).delete()
+
+        with self.verify_action() as events:
+            check_update_all_streams_active_status()
+
+        check_stream_update("events[0]", events[0])
```

Comment:
This test's comments should probably explain a bit more fully that it's testing the transition from True to False via there no longer being any emssages?

Reply:
Done.